src/main/java/com/tasksprints/auction/common/jobrunr/AuctionJob.java

```diff
@@ -0,0 +1,37 @@
+package com.tasksprints.auction.common.jobrunr;
+
+import com.tasksprints.auction.auction.domain.entity.Auction;
+import com.tasksprints.auction.auction.domain.event.AuctionClosedEvent;
+import com.tasksprints.auction.auction.exception.AuctionNotFoundException;
+import com.tasksprints.auction.auction.infrastructure.AuctionRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.context.ApplicationEventPublisher;
+import org.springframework.stereotype.Component;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.math.BigDecimal;
+
+@Component
+@RequiredArgsConstructor
+public class AuctionJob {
+    private final AuctionRepository auctionRepository;
+    private final ApplicationEventPublisher applicationEventPublisher;
+
+    // 경매 시작 상태 변경
+    @Transactional
+    public void startAuction(Long auctionId) {
+        Auction auction = auctionRepository.findById(auctionId).orElseThrow(() -> new AuctionNotFoundException("Not Found Auction ID: " + auctionId));
+        auction.activate();
+        System.out.println("경매 시작 - ID: " + auctionId);
+    }
+
+    // 경매 종료 상태 변경
+    @Transactional
+    public void endAuction(Long auctionId) {
+        Auction auction = auctionRepository.findById(auctionId).orElseThrow(() -> new AuctionNotFoundException("Not Found Auction ID: " + auctionId));
+        auction.close();
+        //잔액 차감 이벤트 요청
+        applicationEventPublisher.publishEvent(new AuctionClosedEvent(auctionId, auction.getHighestBidderId(), auction.getHighestBidAmount()));
```

Comment:
```suggestion
         AuctionClosedEvent event = new AuctionClosedEvent(auctionId, highestBidderId, highestBidAmount);
        applicationEventPublisher.publishEvent(event);
```
처럼 분리를 해도 좋을 것 같습니다. 이벤트 처리 로직 자체를 메소드 단위로 분리하면 조금 더 비즈니스 코드가 더 명확할 것 같습니다.